peerprep-fe/src/lib/socket.ts

```diff
@@ -0,0 +1,35 @@
+// pages/api/socket.js
+
+import { WebSocketServer } from 'ws';
+
+export default function handler(req: any, res: any) {
```

Comment:
Here as well, the styles a bit off

Reply:
hmm, my bad this entire file isnt supposed to be here LOL. removign this